esmvaltool/config-references.yml

```diff
@@ -77,11 +77,12 @@ authors:
     name: Aerts, Jerom
     institute: University of Delft, Netherlands
     email: J.P.M.Aerts@tudelft.nl
```

Comment:
E-mail is given only for the core development team members, because keeping this information up-to-date for the whole team is not feasible.